src/directives/resizable/ResizableTable.ts

```diff
@@ -77,25 +83,52 @@ export default class ResizableTable {
     const rows: HTMLCollection = table.getElementsByTagName('tr');
     return rows[0].children;
   }
+
+  // Add some more px to display chars if some are cropped
+  adaptWidthToContent(colElement: HTMLElement, index: number): number {
+    const contentChars = this.options.columns[index].toString().length;
+    if (!this.options.labelClass || contentChars <= this.options.charsPerCol)
+      return colElement.offsetWidth;
+
+    // Retrieve the label DOM element
+    const labelElement = colElement.getElementsByClassName(
+      this.options.labelClass,
+    )[0] as HTMLElement;
+    if (!labelElement) return colElement.offsetWidth;
+    // Find the padding width in total col width
+    const padding = colElement.offsetWidth - labelElement.offsetWidth;
+    // Find the pixel width for a char based on label width and actual chars per col
+    const pixelPerChar = labelElement.offsetWidth / this.options.charsPerCol;
+    // Calculate needed width for col based on chars in content
+    const contentWidth = pixelPerChar * contentChars;
+    // Calculate max available width for col based on max chars per col
+    const maxWidth = pixelPerChar * this.options.maxCharsPerCol;
+    return padding + Math.ceil(contentWidth < maxWidth ? contentWidth : maxWidth);
```

Comment:
Logic is fine IMHO.
Here is just a little suggestion "pour la route" :wink: 

```suggestion
    return padding + Math.ceil(Math.min(contentWidth, maxWidth));
```

Reply:
yes right ;)